spring-cloud-gcp-samples/spring-cloud-gcp-secretmanager-sample/src/main/resources/bootstrap.properties

```diff
@@ -1,7 +0,0 @@
-# Configuration-level settings for Secret Manager must be specified in a bootstrap.properties file because
```

Comment:
`bootstrap.properties` is referenced in many places in the documentation. Does it need to be updated?

Reply:
The docs of secret manager are updated and `bootstrap.properties` is deleted.